features/192-asset-gear/index.md

```diff
@@ -0,0 +1,667 @@
+# Asset Gear
+
+## Goals
+
+- Store user-generated asset to fulfill application needs.
+  - Photos and short videos taken by the application.
+  - Images and short videos chosen from the library.
+  - Artwork by the user.
+  - Attachments in a chat application.
+  - Documents of reasonable size.
+- Process image asset on the fly via query string.
```

Comment:
Ummm, I think the feature of process image asset is needed for static website hosting, implementation-wise good to think about how to avoid duplicated code? (Or even reconsider if the two gears should be merged?)

Reply:
The image processing code should be highly reusable. However I still prefer the two gears instead of one.